src/components/akyo-detail-modal.tsx

```
@@ -103,7 +103,30 @@
+  // 三面図（PNG）優先、WebPフォールバック
+  const r2Base = process.env.NEXT_PUBLIC_R2_BASE || 'https://images.akyodex.com';
+  const pngUrl = `${r2Base}/${localAkyo.id}.png`;  // 三面図（優先）
+  const webpUrl = buildAvatarImageUrl(localAkyo.id, localAkyo.avatarUrl, 800);  // サムネイル（フォールバック）
+  const [imageUrl, setImageUrl] = useState(pngUrl);
+  const [imageLoadAttempt, setImageLoadAttempt] = useState(0);
+
+  // PNG→WebPフォールバック処理
+  const handleImageError = useCallback(() => {
+    if (imageLoadAttempt === 0) {
+      // PNG失敗 → WebPにフォールバック
+      console.log(`[detail-modal] PNG not found for ${localAkyo.id}, falling back to WebP`);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)